src/shared/components/case-editor/services/wizard-page-field-to-case-field.mapper.ts

```diff
@@ -64,6 +66,33 @@ export class WizardPageFieldToCaseFieldMapper {
     }
   }
 
+  private fixShowConditionPath(caseField: CaseField, pathPrefix: string) {
+    if (caseField.show_condition && pathPrefix.length > 0 && !caseField.show_condition.startsWith(pathPrefix)) {
+      caseField.show_condition = pathPrefix.length > 0 ? pathPrefix + '.' + caseField.show_condition : caseField.show_condition;
```

Comment:
do we need 'pathPrefix.length > 0' check again as same check is done in the if block?

Reply:
👍